clients/fides-js/src/fides-tcf.ts

```diff
@@ -96,6 +99,9 @@ declare global {
       callback: (tcData: TCData, success: boolean) => void,
       parameter?: number | string
     ) => void;
+    config: {
+      fides: OverrideOptions;
+    };
```

Comment:
This is specifically what some applications need to support -  a global object on `window` that they can prefill with some options for various other JS to pull from. It can't be namespaced just to Fides as it's shared with other tools.

It'd be good to support an explicit config option for `init` that names this though, like
```
Fides.init(extra: window.config.fides)
// or
Fides.init(extra: window.options)
// or
Fides.init(extra: window.my.special.options
```
?

Reply:
hmm using `window.Fides` won't currently work as we overwrite `window.Fides` during init.. But I can probably update our code to check for existing `window.Fides.overrideOptions` first, as long as @NevilleS you're OK with changing this?

Reply:
> This is specifically what some applications need to support - a global object on `window` that they can prefill with some options for various other JS to pull from. It can't be namespaced just to Fides as it's shared with other tools.
> 
> It'd be good to support an explicit config option for `init` that names this though, like
> 
> ```
> Fides.init(extra: window.config.fides)
> // or
> Fides.init(extra: window.options)
> // or
> Fides.init(extra: window.my.special.options
> ```
> 
> ?

ah okay—I like that idea! I know we want to ship this portion out soon though, so maybe we should make the "configurability of the config" a follow up ticket, and leave this part as-is? Up to @eastandwestwind !

Reply:
Since we already support Fides.init() to be called with explicit extra options, if we add a "named" option using `Fides.init(extra: window.config.Fides)`, then we'd still keep the ability to pass those extras in `options`, like this, right?

```
Fides.init({options: {fidesEmbed: true}}, extra: window.config.Fides)
```

Reply:
Created a ticket to further discuss- https://github.com/ethyca/fides/issues/4304